blueflood-http/src/integration-test/java/com/rackspacecloud/blueflood/inputs/handlers/HttpHandlerIntegrationTest.java

```diff
@@ -480,4 +484,88 @@ public void testMultiTenantPartialFailureWithoutTenant() throws Exception {
     private ErrorResponse getErrorResponse(HttpResponse response) throws IOException {
         return new ObjectMapper().readValue(response.getEntity().getContent(), ErrorResponse.class);
     }
+
+    @Test
+    public void testIngestWithContentTypeNonJsonShouldReturn415() throws Exception {
+
+        HttpResponse response = httpPost(TENANT_ID, postPath, "random text", ContentType.APPLICATION_SVG_XML);
+        assertEquals("content-type non Json should get 415", 415, response.getStatusLine().getStatusCode());
```

Comment:
In this case, we are returning text, because the HTTP Post does not specify Accept header. By default, if Accept header is not specified, it is assumed ```*/*```.